invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/DocumentMetadataAccordion.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { Accordion, Divider, Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { DocumentRelations, DocumentInfo } from './components';
+import { DocumentTableOfContent } from './components/DocumentTableOfContent';
+import { DocumentConference } from './components/DocumentConference';
+
+export default class DocumentMetadataTabs extends Component {
+  constructor(props) {
+    super(props);
+    this.document = props.documentDetails;
+  }
+
+  state = { activeIndex: 0 };
+
+  handleClick = (e, titleProps) => {
+    const { index } = titleProps;
+    const { activeIndex } = this.state;
+    const newIndex = activeIndex === index ? -1 : index;
```

Comment:
Maybe the indexes can be explicit Strings i.e. `Details` so if we change the order there won't be confusion

Reply:
👍 I will change it with my next PR